MAINTAINERS.yml

```diff
@@ -103,11 +104,16 @@
 #
 # Changes to MAINTAINERS.yml need to be approved as follows:
 #
+#     * Adding a new area needs approval from the Technical Steering Committee.
+#
 #     * Changing the 'maintainers' for an area needs approval from the
-#       Technical Steering Committee
+#       Technical Steering Committee.
+#
+#     * Changing the 'collaborators' lines requires approval of the maintainers and
```

Comment:
minor: Beware this could be understood as requiring "all" collaborators to approve.